README.md

```diff
@@ -1,17 +1,59 @@
-# React Mouse Events
+# React Events in detail
 
 ## Objectives
 
-1. Describe how we might use mouse events to enhance interactivity
-2. Describe how we can attach and use click events
+1. Describe how we can use event data
+2. Describe how event pooling works
 
-## Overview
+## Accessing event data
+Let's take a deeper look at the actual event being passed through. A `SyntheticEvent` event has all of its usual
+properties and methods. This includes its type, target, mouse coordinates, and so on. As a reminder, we add an event
```

Comment:
"This includes..." -> "These include..." (Since the pronoun refers to the plural "properties and methods".)